packages/perseus/src/widgets.ts

```diff
@@ -141,6 +142,12 @@ export const getWidgetScorer = (name: string): WidgetScorerFunction | null => {
     return widgets[name]?.scorer ?? null;
 };
 
+export const getWidgetPublicOptionsFunction = (
+    name: string,
+): PublicWidgetOptionsFunctions | null => {
+    return widgets[name]?.getPublicOptions ?? null;
+};
+
```

Comment:
We talked about adding this, so I threw it in, but not sure if there's somewhere I need to add it yet. I think this will be used on the assessment editor side, but not positive. Although I know Matthew is splitting up the widget exports so maybe we don't need this right now.